### PR TITLE
Modifying xDSCResourceDesigner so it does not load Add-Type twice

### PR DESCRIPTION
**Pull request by tysonjhayes**
This has been breaking my Pester tests that are using this module if for some reason I need to remove it. Basically its checking if the Type already exists if it doesn't then it loads it.

### CODE REVIEW

**Comment by tysonjhayes** [comment]
Would anyone like to give me some feedback on this one?

**Comment by tysonjhayes** [comment]
@TravisEz13 @vors would one of you be so kind as to give me some feedback?

File: xDSCResourceDesigner.psd1 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '1.4.0.0'
+ModuleVersion = '1.4.0.1'
```
Comment (vors) [inline xDSCResourceDesigner.psd1]: please, leave it as is

**Comment by vors** [comment]
LGTM, except commit about [version number](https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#creating-a-new-dsc-resource-in-an-existing-module). Please revert it.

File: xDSCResourceDesigner.psd1 (hunk 1 above)
Comment (tysonjhayes) [inline xDSCResourceDesigner.psd1]: Can do, I don't understand why though.

Comment (vors) [inline xDSCResourceDesigner.psd1]: versions must correspond to releases in the PSGallery. It also would help reduce merge conflicts among other contributors.

**Comment by vors** [comment]
Can you also include this changes to [README.md](README.md) for the future release?
like this: 

```markdown
## Versions

### Unreleased

-  Fix problems with calling Add-Type twice.
```

It would simplify our work, when we will do the next release (last time @KarolKaczmarek hunted in git history to generate version changelog).

**Comment by tysonjhayes** [comment]
Is there anything more we need to merge this?

**Comment by KarolKaczmarek** [comment]
LGTM

**Pull request merged**